index.html

```diff
@@ -248,12 +248,26 @@ <h3 class="hed-subsection">Picturefill function options</h3>
 				</li>
 			</ul>
 
+			<h3 class="hed-subsection">Picturefill plugins/extensions</h3>
+
+			<p>picturefill core polyfills the core part of responsive images including full support for <code>srcset</code>, <code>sizes</code> and <code>picture</code>/<code>source</code>.</p>
+
+			<p>Some aspects of the native feature are left out to provide a clean and lightweight polyfill.</p>
+
+			<p>In case you need a specific behavior of the native implementation, that is not included in picturefill core, just add the corresponding plugin. The best way is to merge all scripts together.</p>
+
+			<ul>
+				<li><strong><a href="dist/plugins/intrinsic-dimension/">intrinsic dimension</a></strong>: This extensions adds support for intrinsic dimension behavior of responsive images. In case you don't style the your images with CSS you probably want this plugin.</li>
+				<li><strong><a href="dist/plugins/mutation/">Mutation plugin</a></strong>: This plugin automatically detects any DOM mutation and polyfills new or changed responsive images automatically. It also adds support for responsive images IDL attributes/properties. If you have a highly dynamic website or a SPA you probably want to use this plugin. (This plugin does not work with IE8.)</li>
```

Comment:
It's "IDL attributes" or "properties". The spec people distinguish between content attributes and idl attributes. While developers normally distinguish between attributes and properties.